sklearn/preprocessing/label.py

```diff
@@ -304,18 +304,41 @@ def fit(self, y):
         self.classes_ = unique_labels(y)
         return self
 
+    def fit_transform(self, y):
+        """Fit label binarizer and transform multi-class labels to binary
+        labels.
+
+        The output of transform is sometimes referred to by some authors as
+        the 1-of-K coding scheme.
+
+        Parameters
+        ----------
+        y : array or sparse matrix of shape (n_samples,) or \
+(n_samples, n_classes)
```

Comment:
Agree that it looks better if it's at the same indent level as the preceding text. My concern is that it renders with spaces between `or` and `(n_samples, n_classes)`. I'll check it when I get home tonight.